growth/too/models.py

```diff
@@ -683,6 +690,8 @@ class Status(enum.Enum):
         nullable=False,
         comment='Plan status')
 
+    event = db.relationship(Event, viewonly=True)
```

Comment:
On master, I added a backref to the other side of the relationship, so this property will be added automatically and you can remove it.